wallet-new/src/Cardano/Wallet/WalletLayer/Types.hs

```diff
@@ -1,20 +1,53 @@
 module Cardano.Wallet.WalletLayer.Types
     ( PassiveWalletLayer (..)
     , ActiveWalletLayer (..)
+    -- * Lenses
+    , pwlCreateWallet
+    , pwlGetWalletIds
+    , pwlGetWallet
+    , pwlUpdateWallet
+    , pwlDeleteWallet
+
+    , pwlCreateAccount
+    , pwlGetAccounts
+    , pwlGetAccount
+    , pwlUpdateAccount
+    , pwlDeleteAccount
+
+    , pwlGetAddresses
```

Comment:
What I like about this layer is that if you squint hard, you will see how is shaping up to essentially be a CRUD layer, because for each resource we do the usual _Create, Read, Update, Delete_. Would be nice to reshuffle the order of the functions (or amend them) to follow this pattern: for each resource, we expose just these 4 operations. I would be curious to see where we get by chasing this trail.